agents-and-function-calling/bedrock-agents/features-examples/11-get-user-confirmation-before-action-execution/11.1-create-agent-with-function-definition-and-user-confirmation/11.1-create-agent-with-function-definition-and-user-confirmation.ipynb

```diff
@@ -0,0 +1,2222 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "089569ae-9348-4442-8e2a-941a8e5c00ac",
+   "metadata": {},
+   "source": [
+    "# Create Agent with Function Definition and User Confirmation\n",
```

Comment:
we should add the architecture diagram of what is being done here with the user confirmation --will share ppt in slack